### ISSUE TITLE
Add some basic statistics with Panda

### ISSUE THREAD

**Issue by ellenkoenig** [OpenTechSchool/python-data-intro#21]
Idea: Add some more statistical processing such as averages, median, percentiles, correlations to follow after the intro to pandas using the "radish voting" example from here: http://opentechschool.github.io/python-data-intro/core/survey.html

**Comment by ghost** [comment]
Would this include an intro to Pandas or would two sections make more sense: "Intro to Pandas and data frames" and "Basic statistics with Pandas"? I tend to lean more towards the bite-sized-chunks strategy. Too much on one page and you lose the overview and get frustrated. I've added a related [issue](https://github.com/OpenTechSchool/python-data-intro/issues/28).

**Comment by ellenkoenig** [comment]
I agree with you, and I in fact tried to say the same thing. I guess I was not clear. The part about "re-doing the radish analysis in pandas"  is supposed to be an intro to pandas and data frames. The intro to stats with pandas would be the next section as I envision it.

**Comment by ellenkoenig** [comment]
I guess splitting the two sections into two issues is a better approach. I will move part of my idea there so that this issue here only covers the stats section.
